### PR TITLE
New Mother And Trees Story

### PR DESCRIPTION
**Pull request by Rai99**
~~Finally I can release more PRs~~

### CODE REVIEW

File: Monika After Story/game/script-stories.rpy [hunk 1]
```diff
@@ -447,6 +447,42 @@ label mas_story_immortal_love:
...
+    m 1eua "He was busy with his life, but he still made time to keep in touch with his mother."
+    m 2ekc "Until one day...{w} he got the call."
+    m 2rksdlc "His mother had died and was found lying next to one of the trees."
+    m 2ekd "In her will, she only had one request of him, and that was to keep taking care of and talking to the trees every day."
+    m 1eka "He took good care of the trees of course, but he still couldn't bring himself to talk to them."
+    m 3euc "Some time later, while he was looking through and cleaning up his mother's old belongings, he found an envelope."
+    m 1eud "Inside, he was shocked at what he found."
+    m 2euc "In it, were three stillborn death certificates for his 'would be' siblings."
+    m 2dsc "He noticed that they each had an identical name to one of the trees that had been in the backyard all his life."
+    m 2dsd "All his life, he had never known that he had siblings, but he finally understood and knew where his mother had buried their remains."
```
Comment (MisterSimple) [inline Monika After Story/game/script-stories.rpy]:
```suggestion
    m 2dsd "He had never known that he had siblings, but he finally understood and knew where his mother had buried their remains."
```

Considering that the last line already ends with "all his life", I think it's not necessary to start this line with it.

File: Monika After Story/game/script-stories.rpy [hunk 2]
```diff
@@ -447,6 +447,42 @@ label mas_story_immortal_love:
...
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_story_mother_and_trees",
+        prompt="A mother and her trees",unlocked=False),code="STY")
+
+label mas_story_mother_and_trees:
+    call mas_story_begin
+    m 1eua "There was once a boy who lived with his mother."
+    m 1hua "She gave him all the affection a mother could give..." 
+    m 3rksdla "But he always thought she could be a bit weird sometimes."
+    m 3eub "Every birthday, she would {i}always{/i} make cookies for him and all his classmades to thank them for being his friends."
```
Comment (MisterSimple) [inline Monika After Story/game/script-stories.rpy]:
```suggestion
    m 3eub "Every birthday, she would {i}always{/i} make cookies for him and all his classmates to thank them for being his friends."
```

File: Monika After Story/game/script-stories.rpy [hunk 3]
```diff
@@ -447,6 +447,42 @@ label mas_story_immortal_love:
...
+    m 2ekc "Until one day...{w} he got the call."
+    m 2rksdlc "His mother had died and was found lying next to one of the trees."
+    m 2ekd "In her will, she only had one request of him, and that was to keep taking care of and talking to the trees every day."
+    m 1eka "He took good care of the trees of course, but he still couldn't bring himself to talk to them."
+    m 3euc "Some time later, while he was looking through and cleaning up his mother's old belongings, he found an envelope."
+    m 1eud "Inside, he was shocked at what he found."
+    m 2euc "In it, were three stillborn death certificates for his 'would be' siblings."
+    m 2dsc "He noticed that they each had an identical name to one of the trees that had been in the backyard all his life."
+    m 2dsd "All his life, he had never known that he had siblings, but he finally understood and knew where his mother had buried their remains."
+    m 1dubsu "He always wanted to take his mother's wish very seriously, and it was then when he started talking to the trees everyday, just as his mother wished."
```
Comment (MisterSimple) [inline Monika After Story/game/script-stories.rpy]:
```suggestion
    m 1dubsu "He always wanted to take his mother's wish very seriously, and it was then when he started talking to the trees every day, just as his mother wished."
```

**Comment by Rai99** [comment]
Untitled713

**Pull request merged**